src/common/runtime/src/global.rs

```diff
@@ -75,13 +71,34 @@ impl GlobalRuntimes {
     define_spawn!(bg);
 
     fn new(read: Option<Runtime>, write: Option<Runtime>, background: Option<Runtime>) -> Self {
+        fn env_rt_size(name: &str) -> usize {
+            std::env::var(name)
+                .ok()
+                .and_then(|v| v.parse().ok())
```

Comment:
I suppose a malformed config deserves a warning or even we should panic.

It's subtle that user configures `GLOBAL_READ_RT_SIZE=8o` or other unintentional typos but we silently ignore it.